src/common/ceph_context.h

```diff
@@ -258,7 +258,13 @@ class CephContext {
   std::string get_set_gid_string() const {
     return _set_gid_string;
   }
-
+  #pragma GCC push_options
+  #pragma GCC optimize ("O0")
+  void leak_some_memory() {
+    volatile char *foo = new char[1234];
+    (void)foo;
+  }
+  #pragma GCC pop_options
```

Comment:
can we put this back to the .cc file? just to improve the compile-time performance?

Reply:
done